From a0e5d11a28c6bc5bcc72927e06b643eb39ea3af3 Mon Sep 17 00:00:00 2001
From: Jeremy Newton <Jeremy.Newton@amd.com>
Date: Wed, 6 Apr 2022 18:37:47 -0400
Subject: [PATCH 3/3] SWDEV-321116 - Drop unnecessary ICD include

This looks unused. This should allow building rocm-opencl without any
dependency on any particular OCL ICD.

Signed-off-by: Jeremy Newton <Jeremy.Newton@amd.com>
Change-Id: Ib9229ede8400fd0e883275659ab99513b03952cb
---
 amdocl/cl_execute.cpp | 2 --
 amdocl/cl_icd.cpp     | 2 --
 2 files changed, 4 deletions(-)

diff --git a/amdocl/cl_execute.cpp b/amdocl/cl_execute.cpp
index 71fe535..e1d5be2 100644
--- a/amdocl/cl_execute.cpp
+++ b/amdocl/cl_execute.cpp
@@ -27,8 +27,6 @@
 #include "platform/program.hpp"
 #include "os/os.hpp"
 
-#include <icd/loader/icd_dispatch.h>
-
 /*! \addtogroup API
  *  @{
  *
diff --git a/amdocl/cl_icd.cpp b/amdocl/cl_icd.cpp
index 2a4f3af..4fce08b 100644
--- a/amdocl/cl_icd.cpp
+++ b/amdocl/cl_icd.cpp
@@ -27,8 +27,6 @@
 #include "cl_d3d11_amd.hpp"
 #endif  //_WIN32
 
-#include <icd/loader/icd_dispatch.h>
-
 #include <mutex>
 
 amd::PlatformIDS amd::PlatformID::Platform =  //{ NULL };
-- 
2.35.1

